build.gradle

```diff
@@ -138,6 +145,9 @@ signing {
 
 
 javadoc {
+  failOnError = false
+  options.addStringOption('Xdoclint:none', '-quiet')
+  options.addBooleanOption('failOnError', false)
```

Comment:
Don't fail the build for public methods missing JavaDocs

Reply:
hmm maybe we should keep that though?

Reply:
I don't know... it was really annoying to have JavaDocs all over the place, such as all the DTO getters and everything and I don't think it was adding much value. _Perhaps_ the auto generated Java Docs help with IDE hints, but we can strive to make sure all public methods exposed to the upstream SDKs have docs or something for those cases. I also think that itself is a stretch, and if we blew away our Java docs today nobody would care as we have our richer Eppo documentation